### ISSUE TITLE
[DOCS] Handling use case where cloud service broker is cycled or updated while performing terraform apply

### ISSUE THREAD

**Issue by xyloman** [cloudfoundry-incubator/cloud-service-broker#170]
Interested to know what happens to the terraform apply process when the cloud service broker is cf pushed, restaged or terminated during a repave of the VM in which its hosted on?  Will the terraform apply be retried or will the cloud service broker stop accepting requests and drain the current ones before terminating gracefully?

**Comment by mogul** [comment]
I found that when a broker instance was restarted due to hitting a memory
limit during a `terraform apply`, the API continues to return the
"operation pending" state forever, and the terraform operation is never
re-attempted. This should definitely be addressed.

**Comment by xyloman** [comment]
yes just noticed this same behavior when attempting to delete a service instance and i believe the broker got turned over in the middle resulting in the same outcome.

**Comment by erniebilling** [comment]
There is currently no support in the broker to handle interrupted operations.

**Comment by mogul** [comment]
Hey folks, any word on plans to address this? We think we ran into it again.

**Comment by pivotal-marcela-campo** [comment]
We are not planning to address this in the close future. Closing this for the moment.
